website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,219 @@
+---
+author: Ayush Sharma
+authorURL: https://twitter.com/ayusharma_
+authorFBID: 1122901551
```

Comment:
Here you need to find your `authorFBID ` :-) otherwise my faces is being displayed.